code-scanning/scorecards.yml

```diff
@@ -0,0 +1,73 @@
+name: Scorecards supply-chain security
+on: 
+  # Only the default branch is supported.
+  branch_protection_rule:
```

Comment:
I'm not sure what the aim is on triggering on branch protection rule changes?

Reply:
We want to provide up-to-date results for the [Branch-Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection) check. Unfortunately the branch protection config is not part of the commit history, so push trigger is not sufficient to provide up-to-date results in the scanning dashboard.